### PR TITLE
#167727614 Configures ESLint for the project

### PR DESCRIPTION
**Pull request by minlinx**
**What does this PR do?**
Configures Eslint for the project
**Description of Task Taken to Complete?**
Installed these development dependencies: **"eslint": "^5.16.0"**,
**"eslint-config-airbnb-base": "^14.0.0"**,
**"eslint-config-prettier": "^6.0.0"**,
**"eslint-plugin-import": "^2.18.2"**
Created a .eslintrc.json file at the project's root directory and set up rules as described on the PT board.
**How should this be manually Tested?**
Clone the repo, run npm install, close your editor and open again.
Check if the Eslint server is running and notice red wiggles for violated rules.

**Link to pivotal tracker story:**
https://www.pivotaltracker.com/n/projects/2381976/stories/167727614

### CODE REVIEW

**Review by 0xciph3r — APPROVED** [review]

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      {
+        "require": {
+          "FunctionDeclaration": true,
+          "MethodDefinition": true,
+          "ClassDeclaration": true
+        }
+      }
+    ]
+  }
+}
```
Comment (Daymorelah) [inline .eslintrc.json]: Add a new line here.

**Review by Daymorelah — CHANGES_REQUESTED** [review]

File: .eslintrc.json (hunk 1 above)
Comment (Daymorelah) [reply .eslintrc.json]: also, add this to the config file
"parserOptions": {
    "ecmaVersion": 6,
    "sourceType": "module",
    "ecmaFeatures": {
        "jsx": true
    }
  },
"settings": {
    "import/resolver": {
      "node": {
        "extensions": [".js"]
      }
    }
  }

**Review by Daymorelah — CHANGES_REQUESTED** [review]

**Review by Daymorelah — COMMENTED** [review]
I cant seem to find a `.eslintignore` file. Did you configure ESLint to lint all files in the codebase?

**Comment by minlinx** [comment]
> I cant seem to find a `.eslintignore` file. Did you configure ESLint to lint all files in the codebase?

I did not see this comment. Just a second please!!!

**Review by Daymorelah — APPROVED** [review]

**Pull request merged**